HW04/Pavel_Mishchenko/spec/mentor_spec.rb

```diff
@@ -0,0 +1,210 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'TestName', surname: 'TestSurname') }
+  let(:mentor) { described_class.new(name: 'Test', surname: 'Testov') }
+  let(:notification1) { Notification.new(title: 'TestTitle1', description: 'TestDescription1') }
+  let(:notification2) { Notification.new(title: 'TestTitle2', description: 'TestDescription2') }
+
+  describe '#notify_student' do
+    context "when method hasn't arguments" do
+      subject { mentor.notify_student }
+
+      it 'raises ArgumentError' do
+        expect { subject }.to raise_error(ArgumentError)
+      end
+    end
+
+    context 'when mentor notifies student' do
```

Comment:
the context should be with opposite meaning

Reply:
you can test here that notifications_list includes the necessary description much easier 
try not to use such huge subject and the matcher `an_object_having_attributes`